app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -388,6 +414,20 @@ private boolean doMongoUpload(SharedPreferences prefs, List<BgReading> glucoseDa
                             BasicDBObject query = new BasicDBObject("type", "cal").append("sysTime", format.format(calRecord.timestamp));
                             dexcomData.update(query, testData, true, false,  WriteConcern.UNACKNOWLEDGED);
                         }
+                        
+                        Log.i(TAG, "The number of sensor records being sent to MongoDB is " + meterRecords.size());
+                        for (Sensor sensor : sensors) {
+                            // make db object
+                            BasicDBObject testData = new BasicDBObject();
+                            testData.put("type", "sensor");
+                            testData.put("xDrip_started_at", sensor.started_at);
+                            testData.put("xDrip_stopped_at", sensor.stopped_at);
+                            testData.put("xDrip_latest_battery_level", sensor.latest_battery_level);
```

Comment:
This will get updated when a new transmitter package comes in with a lower battery level.
I guess it would suffice adding the whole Sensor to the SensorSendQueue again since we switched to upserts.